code/obj/decal/poster.dm

```diff
@@ -678,7 +678,8 @@
 					5 ;"contest-other4",
 					5 ;"contest-other5",
 					5 ;"contest-other6",
-					5 ;"contest-other7"
+					5 ;"contest-other7",
+					5 ;"contest-other8"
```

Comment:
This is for contest posters I think. poster_rand may be the place?

Reply:
good point, ill change that a tad bit later